### ISSUE TITLE
Mask RCNN MSCOCO pretrained checkpoint related question

### ISSUE THREAD

**Issue by Allamrahul** [anirudh-chakravarthy/ObjProp#5]
@anirudh-chakravarthy , I have a question and I hope you can answer it: In the masktrackrcnn git repo (https://github.com/youtubevos/MaskTrackRCNN), they have used MaskRCNN-resnet50-FPN starting with MSCOCO pretrained checkpoint (https://s3.ap-northeast-2.amazonaws.com/open-mmlab/mmdetection/models/mask_rcnn_r50_fpn_1x_20181010-069fa190.pth). But we are unable to access this link.  Could you let us know how we could get access to this checkpoint?

**Comment by anirudh-chakravarthy** [comment]
Hi,

I had previously used that link and I'm not sure why it's not working. However, I've also previously used pre-trained checkpoints from the MMDetection repo for Mask R-CNN: https://github.com/open-mmlab/mmdetection/tree/master/configs/mask_rcnn. Let me know if this helps!
